.github/workflows/codeql.yml

```diff
@@ -0,0 +1,51 @@
+name: "CodeQL"
+
+on:
+  push:
+    branches:
+      - "main"
+  pull_request:
+    branches:
+      - "main"
+  schedule:
+    - cron: '25 4 * * 4'
```

Comment:
Just curious, what is the plan for handling when these scheduled runs fail?

Reply:
you need to check https://github.com/chainguard-dev/go-apk/security/code-scanning from time to time